frontend/packages/volto-intranet/package.json

```diff
@@ -53,7 +53,7 @@
   ],
   "dependencies": {
     "@kitconcept/core": "workspace:*",
-    "@kitconcept/volto-solr": "2.0.0-alpha.1",
+    "@kitconcept/volto-solr": "2.0.0-alpha.2",
```

Comment:
@ericof this commit made all a11y tests fail. I have no clue why this happens.

Reply:
@tisto This is the rule that is failing: "Ensures every ARIA input field has an accessible name (aria-input-field-name)"

I guess it is something wrong with the autocomplete livesearch input from https://github.com/kitconcept/kitconcept.solr/pull/43

Reply:
> @ericof this commit made all a11y tests fail. I have no clue why this happens.

This is the latest released version of `@kitconcept/volto-solr`. We need to fix the issue there :-(

Reply:
@davisagli right. Good catch. The good thing is that it means we fetch the correct code and functionality. Then someone needs to look at how we can fix these a11y regressions.

Reply:
<img width="1507" height="234" alt="Screenshot 2025-09-19 at 3 42 07 PM" src="https://github.com/user-attachments/assets/da56ea75-2133-45d4-864b-d0ea1231d1ea" />

Reply:
kitconcept.intranet is passing an aria-label here:

https://github.com/kitconcept/kitconcept.intranet/pull/189/files#diff-f5cb42803c283b5e1d30960373e13f4973dcfa47f4c70f45876aee2deca38b21R168

but it is not used by the SolrSearchAutosuggestWidget in kitconcept-solr (https://github.com/kitconcept/kitconcept.solr/blob/main/frontend/packages/volto-solr/src/components/theme/SolrSearchWidget/SolrSearchWidget.jsx)

Reply:
@tisto @ericof I fixed a conditional so that it won't try to use the autosuggest search widget when solr is not active. That should be enough to make the a11y tests pass. We still need to follow up in kitconcept.solr and fix the issue so that the aria-label is there when it _is_ enabled. But at least this way we won't cause a regression for sites that aren't using solr.

Reply:
I also added an issue: https://github.com/kitconcept/kitconcept.solr/issues/47